OpenRA.Mods.Common/AI/States/NavyStates.cs

```diff
@@ -30,12 +30,12 @@ protected Actor FindClosestEnemy(Squad owner)
 			// (Way better than finding a nearest target which is likely to be on Ground)
 			// You might be tempted to move these lookups into Activate() but that causes null reference exception.
 			var domainIndex = first.World.WorldActor.Trait<DomainIndex>();
-			var mobileInfo = first.Info.TraitInfo<MobileInfo>();
-			var passable = (uint)mobileInfo.GetMovementClass(first.World.Map.Rules.TileSet);
+			var locomotorInfo = first.Trait<Mobile>().Info.LocomotorInfo;
```

Comment:
Why change from `first.Info.TraitInfo<MobileInfo>()` to `first.Trait<Mobile>().Info`?